docs/source/using_doctr/using_models.rst

```diff
@@ -34,45 +34,45 @@ The following architectures are currently supported:
++----------------+---------------------------------+-----------------+---------------+------------+---------------+------------+---------------+--------------------+
+| TensorFlow     | db_resnet50                     | (1024, 1024, 3) | 25.2 M        | 84.39      | 85.86         | 93.70      | 83.24         | 1.2                |
++----------------+---------------------------------+-----------------+---------------+------------+---------------+------------+---------------+--------------------+
+| TensorFlow     | db_mobilenet_v3_large           | (1024, 1024, 3) | 4.2 M         | 80.29      | 70.90         | 84.70      | 67.76         | 0.5                |
++----------------+---------------------------------+-----------------+---------------+------------+---------------+------------+---------------+--------------------+
+| TensorFlow     | linknet_resnet18                | (1024, 1024, 3) | 11.5 M        | 81.37      | 84.08         | 85.71      | 83.70         | 0.7                |
++----------------+---------------------------------+-----------------+---------------+------------+---------------+------------+---------------+--------------------+
+| TensorFlow     | linknet_resnet34                | (1024, 1024, 3) | 21.6 M        | 82.20      | 85.49         | 87.63      | 87.17         | 0.8                |
++----------------+---------------------------------+-----------------+---------------+------------+---------------+------------+---------------+--------------------+
+| TensorFlow     | linknet_resnet50                | (1024, 1024, 3) | 28.8 M        | 80.70      | 83.51         | 86.46      | 84.94         | 1.1                |
++----------------+---------------------------------+-----------------+---------------+------------+---------------+------------+---------------+--------------------+
+| TensorFlow     | fast_tiny                       | (1024, 1024, 3) | 13.5 M (8.5M) |            |               |            |               | 0.7 (0.4)          |
++----------------+---------------------------------+-----------------+---------------+------------+---------------+------------+---------------+--------------------+
+| TensorFlow     | fast_small                      | (1024, 1024, 3) | 14.7 M (9.7M) |            |               |            |               | 0.7 (0.5)          |
++----------------+---------------------------------+-----------------+---------------+------------+---------------+------------+---------------+--------------------+
+| TensorFlow     | fast_base                       | (1024, 1024, 3) | 16.3 M (10.6M)|            |               |            |               | 0.8 (0.5)          |
++----------------+---------------------------------+-----------------+---------------+------------+---------------+------------+---------------+--------------------+
```

Comment:
What are the values in parenthesis ?

Reply:
The values from the reparameterized model 

It's smaller and much faster by using the inference only reparameterized version